acquia/acquia_cms
# ACMS-647: Added the gdpr modules.

**Motivation**
* Added a new module by the name Acquia CMS privacy for GDPR and EU cookie compliance.
Fixes #NNN

**Proposed changes**
* Added a new module by the name Acquia CMS privacy for GDPR and EU cookie compliance.

**Alternatives considered**
* None

**Testing steps**
* Head towards the module install page and verify that the Acquia CMS Privacy, GDPR and EU cookie compliance modules are enabled.
* The configuration can be verified on Drupal Configuration pages.
<img width="1791" alt="Screenshot 2021-10-20 at 6 50 09 PM" src="https://user-images.githubusercontent.com/15887127/138100821-a06ed6c4-fb69-4564-91d4-5c94e7967866.png">


**Merge requirements**
